src/enforcer/HolographRoyalties.sol

```diff
@@ -324,11 +319,16 @@ contract HolographRoyalties is Admin, Owner, Initializable {
     uint256 balance = erc20.balanceOf(address(this));
     require(balance > 10000, "ROYALTIES: Not enough tokens");
```

Comment:
Same thing here

Reply:
Need to remove it from here since some tokens are using no decimals or two decimal places, this would result in $100 for something like USDT getting stuck in account. I think there was a specific C4 issues associated with this one.

Reply:
I removed this one